plugins/CoreHome/vue/src/createAngularJsAdapter.ts

```diff
@@ -64,10 +72,21 @@ function toAngularJsCamelCase(arg: string): string {
     .replace(/-([a-z])/g, (s, p) => p.toUpperCase());
 }
 
+export function processScopeProperty<T>(newValue: T): T {
+  if (typeof newValue === 'object'
+    && newValue !== null
+    && Object.getPrototypeOf(newValue) === Object.prototype
+  ) {
+    return Object.fromEntries(Object.entries(newValue).filter((pair) => !/^\$/.test(pair[0]))) as T;
+  }
+
+  return newValue;
+}
```

Comment:
AngularJS sometimes adds its own properties to scope properties (like `$$hashKey`), this removes them when sending them to Vue.